doc/mainpage.txt

```diff
@@ -0,0 +1,10 @@
+/**
+\mainpage API Documentation
+
+\section intro_sec Introduction
+
+Documentation of the Ledger API is an ongoing process and you are invited
+to help out and contribute. In case you find this documentation incorrent,
+incomplete, unclear, or entirely lacking please
+[open a pull request](https://git.ledger-cli.org/ledger/pulls).
+*/
```

Comment:
Over time more information about contributing to or programming with ledger should be added to the API documentation's main page.